src/index_runner/es_indexers/main.py

```diff
@@ -61,11 +61,13 @@ def index_obj(obj_data, ws_info, msg_data):
     defaults = indexer_utils.default_fields(obj_data, ws_info, obj_data_v1)
     for indexer_ret in indexer(obj_data, ws_info, obj_data_v1):
         if indexer_ret['_action'] == 'index':
-            if config()['allow_indices'] and indexer_ret.get('index') not in config()['allow_indices']:
+            allow_indices = config()['allow_indices']
```

Comment:
Changed it so `config()['allow_indices']` can be `None` if not set, so had to tweak the code here.